nearswap/src/lib.rs

```diff
@@ -791,7 +792,7 @@ mod tests {
         let mut ctx = Ctx::new(vec![], false);
         ctx.vm.attached_deposit = attached_near;
         testing_env!(ctx.vm.clone());
-        let contract = NearCLP::new(ctx.accounts.owner.clone());
+        let contract = NearCLP::new("clp_owner".try_into().unwrap());
```

Comment:
Instead of `.try_into().unwrap()` we can do `.into()`

Reply:
ok

Reply:
![trait](https://user-images.githubusercontent.com/26249903/109804662-82454780-7c48-11eb-92f6-e721dc6a0522.png)

getting this error if i use `into`

Reply:
Ah, ok. Your implementation is OK. Otherwise we could also do a helper method (`strToAcc()`).

Reply:
ok keeping this same.